src/server/controllers/VerificationController-util.ts

```diff
@@ -173,6 +174,10 @@ export type ContractWrapper = ContractMeta & {
   contract: CheckedContract;
 };
 
+type Mutable<Type> = {
+  -readonly [Key in keyof Type]: Type[Key];
+};
+
```

Comment:
What's this for?

Reply:
This is used to remove the readonly keyword from the type, needed because we need to update that field in `constructorArgumentsArray`